services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/Application.kt

```diff
@@ -14,5 +14,5 @@ data class Application(
     val applicationIdentifier: String?,
     val applicationType: ApplicationType,
     val applicationData: ApplicationData,
-    val hankeTunnus: String,
+    val hankeTunnus: String?,
```

Comment:
I added a new data class for this to handle cable reports without hanketunnus in this new feature. Application class is left as it was before this pr.

Reply:
I like the new class, but seeing it now, it could be a highly specific DTO for creating the cable report application, like you said in the other thread. 

Specifically, these fields are set to null when creating an entity from the request, so they could be omitted.
```
    val alluid: Int?,
    val alluStatus: ApplicationStatus?,
    val applicationIdentifier: String?,
```

Reply:
Modified as suggested.